### ISSUE TITLE
18. 상품-카테고리 자동 매칭 개발

### ISSUE THREAD

**Issue by Kang-heesuk** [NAVER-CAMPUS-HACKDAY/common#17]
## 주제 : 상품-카테고리 자동 매칭 개발
!본 주제의 우수참가자로 선정시 'NAVER'에서 '채용전환형 인턴십'의 참여기회가 주어집니다.

### 주제선정 배경
* 업무 효율화를 위해 AI를 기반으로 한 상품 카테고리(분류) 자동 매칭 시스템이 필요

### 개발 요구사항 (필수)
* tensorflow, keras 등 오픈소스 프레임워크를 활용하여 Deep learning으로 구현
* 상품명, 브랜드, 제조사 등 텍스트로 구성된 상품데이터를 입력으로 하여, 해당 상품의 카테고리를 분류하는 알고리즘 구현
* 미리 만들어진 정답셋이 활용할 수 있으며, test set 기준 accuracy 80% 달성을 목표로 함

### 개발 요구사항 (선택)
* 모델 설계시 문서화 
* 모델 검증 코드/프로세스 구현시 플러스 포인트

### 개발언어 
* Python 등

### 플랫폼
* NSML(미정-네이버 gpu 플랫폼), Tensorflow, Keras등 AI 플랫폼

### 기타사항
* 개발환경
  - 노트북
  -  tensorflow, keras 등 오픈소스 프레임워크를 활용하여 Deep learning으로 구현
* 참고 
  - test  set기반으로 80%이상의 정확도 확보
